site/operational-guide/autoscaling.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Autoscaling
+menu_order: 40
+---
+An autoscaling configuration begins with a small fixed cluster:
+
+* Configured as per the [Uniform Fixed Cluster](/site/operational-guide/uniform-fixed-cluster.md)
+  scenario
+* Hosted on reserved or protected instances to ensure long-term
+  stability
+* Ideally sized at a minimum of three or five nodes (you can make your
+  fixed cluster bigger to accommodate base load as required; a minimum
+  is specified here in the interests of resilience only)
+
+Building on this foundation, arbitrary numbers of dynamic peers can be
+added or removed concurrently as desired, without requiring any
+changes to the configuration of the fixed cluster.
```

Comment:
Unlike the "Manual/Incremental Deployment" section, this doesn't say what happens on reboot.